src/library/Link/styled.js

```diff
@@ -37,6 +37,10 @@ export const Link = styled('a', {
     // https://developer.mozilla.org/en-US/docs/Web/CSS/:active
     '&:active': {
       color: theme.Link_color_active
-    }
+    },
+
+    /* TargetX Custom Styles */
+    cursor: 'pointer',
+    ...(underline && { textDecoration: 'underline' })
```

Comment:
`underline` is a shorthand for `textDecoration: 'underline'`, so, based on our pattern, please add support for `textDecoration` passthrough prop as well.